tilings/tiling.py

```diff
@@ -181,7 +181,9 @@ def _remove_empty_rows_and_cols(self) -> None:
         """Remove empty rows and columns."""
         # Produce the mapping between the two tilings
         if not self.active_cells:
-            self._forward_map: Dict[Cell, Cell] = {}
+            self._forward_map: Dict[Cell, Cell] = {
```

Comment:
Why do we need to map the empty cell?

Reply:
I recall the logic of adding this was related to the fact that when we map assumptions, the assumption on the parent is not always aware if a cell is actually empty, and so when we map if forward we end up with a key error.

I remove this change and nothing is breaking in our successes, but I'm not 100% sure.

Reply:
My concern came from the fact that it is the opposite of what I did to fix to row col separation bug. It that case I took extra care to not include empty cell in maps to avoid trouble when composing maps. I guess we will see if a bug comes out.